specification/healthdataaiservices/data-plane/HealthDataAIServices.Deidentification/preview/2024-01-31-preview/HealthDataAIServices.Deidentification.json

```diff
@@ -0,0 +1,1309 @@
+            "description": "Vehicle IDs, such as license plates or VIN numbers"
+          },
+          {
+            "name": "Zip",
+            "value": "Zip",
+            "description": "Zip Code."
+          }
+        ]
+      }
+    },
+    "PhiEntity": {
+      "type": "object",
+      "description": "Phi Entity tag in the input.",
+      "properties": {
+        "category": {
+          "$ref": "#/definitions/PhiCategories",
+          "description": "Phi Type of the entity."
+        },
+        "offset": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Starting index of the location from within the input text."
+        },
+        "length": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Length of the input text."
+        },
+        "text": {
+          "type": "string",
+          "description": "Text of the entity."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "Confidence score of the text/type pairing."
+        }
+      },
+      "required": [
+        "category",
+        "offset",
+        "length"
+      ]
+    },
+    "PhiTaggerResult": {
+      "type": "object",
+      "description": "Result of the tag operation.",
+      "properties": {
+        "entities": {
+          "type": "array",
+          "description": "List of entities detected in the input.",
+          "items": {
+            "$ref": "#/definitions/PhiEntity"
+          },
+          "x-ms-identifiers": []
+        },
+        "stringIndexType": {
+          "$ref": "#/definitions/StringIndexType",
+          "description": "Requested Encoding of the tag response indices.",
+          "default": "TextElement_v8"
+        },
+        "path": {
+          "type": "string",
+          "description": "Path to the file in the storage container.",
+          "maxLength": 1024
+        },
+        "checksum": {
+          "type": "string",
+          "description": "Checksum of the file."
+        }
+      },
+      "required": [
+        "entities",
+        "stringIndexType"
+      ]
+    },
+    "SourceStorageLocation": {
+      "type": "object",
+      "description": "Storage location.",
+      "properties": {
+        "location": {
+          "type": "string",
+          "format": "uri",
+          "description": "URL to storage location. Must be a valid Azure Storage SAS URI."
+        },
+        "prefix": {
+          "type": "string",
+          "description": "Prefix to filter files by."
+        },
+        "delimiter": {
+          "type": "string",
+          "description": "Delimiter of the storage paths.",
+          "default": "/"
+        },
+        "recursive": {
+          "type": "boolean",
+          "description": "Recursive flag to indicate if all files in the prefix should be included. Only available as source.",
+          "default": true
+        },
```

Comment:
"recursive" is required and cannot have a default

Reply:
Removed.